draft-ietf-ppm-dap.md

```diff
@@ -1916,9 +1916,9 @@ following checks:
 
 1. Finally, if an Aggregator cannot determine if an input share is valid, it
    MUST mark the input share as invalid with error `report_dropped`. For
-   example, if the Aggregator has evicted the state required to perform the
-   check from long-term storage. (See {{reducing-storage-requirements}} for
-   details.)
+   example, the report timestamp may be so far in the past that the Aggregator
+   has evicted the state required to perform the check from long-term storage.
+   (See {{reducing-storage-requirements}} for details.)
```

Comment:
We can reword to make it less ambiguous what "from long-term storage" applies to. Also the last sentence doesn't need to be in a parenthetical; let your text roam wild and free!
```suggestion
   example, the report timestamp may be so far in the past that the state required to perform the check has been evicted from the Aggregator's long-term storage. See {{reducing-storage-requirements}} for details.
```

Sorry this isn't line wrapped properly.